frontend/src/scenes/PreflightCheck/PreflightCheck.tsx

```diff
@@ -1,302 +1,220 @@
-    status === true -> Item ready (validated)
-    */
-    let textColor: string
-    const { preflightLoading } = useValues(preflightLogic)
-
-    if (status) {
-        textColor = 'var(--success)'
-    } else if (status === false) {
-        if (failedState === 'warning') {
-            textColor = 'var(--warning)'
-        } else if (failedState === 'not-required') {
-            textColor = 'var(--border-dark)'
-        } else {
-            textColor = 'var(--danger)'
-        }
-    } else {
-        textColor = 'var(--border-dark)'
+function PreflightCheckIcon({ status, loading }: { status: PreflightCheckStatus; loading?: boolean }): JSX.Element {
+    const size = {
+        height: '20px',
+        width: '20px',
     }
-
-    const icon = (): JSX.Element => {
-        if (preflightLoading) {
-            return <LoadingOutlined style={{ fontSize: 20, color: textColor }} />
-        }
-        if (status) {
-            return <CheckSquareFilled style={{ fontSize: 20, color: textColor }} />
-        } else {
-            if (failedState === 'warning') {
-                return <WarningFilled style={{ fontSize: 20, color: textColor }} />
-            } else {
-                return <CloseSquareFilled style={{ fontSize: 20, color: textColor }} />
-            }
-        }
+    if (loading) {
+        return <LoadingOutlined style={{ color: 'var(--primary)' }} />
+    }
+    if (status === 'validated') {
+        return <IconCheckCircleOutline {...size} />
+    } else if (status === 'warning' || status === 'optional') {
+        return <IconWarningAmber {...size} />
     }
+    return <IconErrorOutline {...size} />
+}
 
+function PreflightItem({ name, status, caption }: PreflightItemInterface): JSX.Element {
+    const { preflightLoading } = useValues(preflightLogic)
     return (
-        <Col span={12} style={{ textAlign: 'left', marginBottom: 16, display: 'flex', alignItems: 'center' }}>
-            {icon()}
-            <span style={{ color: textColor, paddingLeft: 8 }}>
-                {name}{' '}
-                {caption && status === false && (
-                    <div data-attr="caption" style={{ fontSize: 12 }}>
+        <div className={clsx('preflight-item', preflightLoading ? 'loading' : status)}>
+            <div className="icon-container">
+                <PreflightCheckIcon status={status} loading={preflightLoading} />
+            </div>
+            <div className="central-text-container">
+                <p className="check-name">{name}</p>
+                {caption && (
+                    <p data-attr="caption" className="text-muted">
                         {caption}
-                    </div>
+                    </p>
                 )}
-            </span>
-        </Col>
+            </div>
+
+            <div className="right-status">
+                <p className="status-text" data-attr="status-text">
+                    {capitalizeFirstLetter(preflightLoading ? 'validating' : status)}
```

Comment:
As a user "Validated" or "Error" are not super relevant messages to me. I'd like to know what was validated really, or even more importantly what errored. I think a simple improvement here may be changing "Validated" to "Up" and "Error" to "Down" for services, and maybe linking to some troubleshooting docs for things that are down. Could result in fewer support inquiries. What do you think?

Reply:
That makes sense to me. I just updated "validated" to be "running" and "error" to be "down".

I had asked the platform team about docs for each check, and unfortunately we don't have docs that cleanly map to these checks. The closest that we have is the trouble shooting docs link. I just moved that higher up on this screen because it's definitely the place someone should go if their having issues.